src/main/java/de/adesso/anki/battle/protocol/commands/ShieldCommand.java

```diff
@@ -0,0 +1,16 @@
+package de.adesso.anki.battle.protocol.commands;
+
+import de.adesso.anki.battle.world.bodies.Vehicle;
+
+public class ShieldCommand extends Command {
+	
+	@Override
+    public void execute(Vehicle vehicle) {
+		// TODO:destroy Rocket, mine
+
+		// amount that the shield absorbs
+		// vehicle.setEnergy(vehicle.getEnergy()+25);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=22&issues=AWKWIFf51B6WA4TguGK_&open=AWKWIFf51B6WA4TguGK_)

[ISSUE_KEY]: AWKWIFf51B6WA4TguGK_